### PR TITLE
P2 separate usersays and response

### PR DESCRIPTION
**Pull request by jingwanhsu**
gogo
Trello cards:
https://trello.com/c/Z8tVKbfK/33-ph2-high-separate-usersays-and-response-to-different-nodes
https://trello.com/c/QbamBB7u/28-ph2-high-allow-to-have-more-than-one-user-says-or-responses

### CODE REVIEW

File: components/UserSaysProperty.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react'
+
+const UserSaysProperty = ({ panel, onSaveUserSaysPropertiesClick, onUserSaysChange }) => (
```
Comment (LaContra) [inline components/UserSaysProperty.js:3]: This component is almost the same as ResponseProperty. Is it possible that they use the same component but with different properties? If you don't have time to refactor it right now, we can do it after the deadline of this project.

File: reducers/cyElements.js [hunk 2]
```diff
@@ -88,22 +136,48 @@ const cyElements = (state, action) => {
...
+        },
+        {
+          group: "edges",
+          data: {source: action.id-3, target: action.id-2, id: action.id-1},
+          classes: "us2r",
+          selectable: false,
+        }
+      ]
       // TODO: position
       // position: { x: -this.state.cy.viewport().pan().x / this.state.cy.zoom() + 40, y: -this.state.cy.viewport().pan().y / this.state.cy.zoom() + 40 }
```
Comment (LaContra) [inline reducers/cyElements.js:157]: I remember you already fix it?

Comment (jingwanhsu) [inline reducers/cyElements.js:157]: What did you mean? If u mean the position, I haven't do anything for that

File: components/UserSaysProperty.js (hunk 1 above)
Comment (jingwanhsu) [inline components/UserSaysProperty.js:3]: Exactly. I thought it once but decided to separate them coz they might be different in the future. There should be an action field on response, right?

File: reducers/cyElements.js (hunk 2 above)
Comment (LaContra) [inline reducers/cyElements.js:157]: Got it

File: components/UserSaysProperty.js (hunk 1 above)
Comment (LaContra) [inline components/UserSaysProperty.js:3]: Got it. I'll merge it right now.

**Pull request merged**